apps/cp-demo-kstreams/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM maven:3.6.1-jdk-8 AS build
+RUN mkdir /build
+WORKDIR /build
+COPY checkstyle.xml .
+COPY pom.xml . 
+COPY src/assembly/ ./src/assembly
+RUN mkdir -p ./src/main/resources/avro/io/confluent/cpdemo
+RUN mvn -B clean dependency:resolve dependency:resolve-plugins dependency:go-offline package
+COPY src/ ./src/
+RUN mvn -B clean package
+ 
+FROM confluentinc/cp-kafka:5.3.0
```

Comment:
@rspurgeon as it currently stands, will we need to manually update this with every new CP release?  If so, can you please talk to the tools team to see if they can augment their automation to detect this and bump it automagically?  (same with the jar name below).

Reply:
Agree, looking into this

Reply:
I'll wait for feedback on this and address in a future PR